packages/grep-workspace/README.md

```diff
@@ -0,0 +1,6 @@
+# Theia - Grep Workspace Extension
```

Comment:
It should not be `grep` extension for sure, it is just implementation details of the search engine, it should be possible to change the implementation without renaming the package.

Reply:
@svenefftinge Should not actually the search functionality belong the workspace package?

Reply:
I am in a very early stage of the implementation. We had internal discussions last week about this. Things are not set in stone for the moment. Thanks for the feedback.

Reply:
I know :) btw have you looked at the code of VS code? Maybe we can learn something from them.

Reply:
e.g. there is the node library for the glob search: https://www.npmjs.com/package/glob
maybe there is similar for regex if we need to support it

Reply:
I think it's fine to have it separated from the workspace extension.

Reply:
@akosyakov  the idea was to have 1 generic package that you can change the implementation and 1 package that contains the implementation.

I think we could replace grep-workspace to search-workspace as the generic package name ?

Reply:
@lmcgupe let's rename the implementation 'ripgrep-workspace' and we'll have the general interface 'search-workspace' 

The class names should follow